content/articles/account-invoice-history.md

```diff
@@ -55,39 +55,31 @@ Depending on your plan, your subscription will renew every month or every year.
 A `dunned` invoice will automatically be retried once per week for three weeks. You can manually [retry the payment](#retrying) any time. We'll notify you via email on every attempt.
```

Comment:
Do we still offer yearly renewals? Have I asked this already?

Reply:
Only for people who are still on a legacy plan (Personal, Professional). 

The Solo and Teams plan are only monthly.